src/include/dstore.h

```diff
@@ -182,4 +182,5 @@ int dstore_pwrite(struct dstore_obj *obj, off_t offset, size_t count,
  */
 int dstore_pread(struct dstore_obj *obj, off_t offset, size_t count,
 		       size_t bs, char *buf);
+
```

Comment:
Reason for adding this new line? Apart from this no other change in this file.

Reply:
I added a declaration here. Later removed it and newline remained. Removed new line.